src/screens/cart/LoginFirstCartScreen/LoginFirstCartScreen.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {FC} from 'react'
+import styles from './styles'
+import {Text, View} from 'react-native'
+import {Button} from 'react-native-elements'
+import {IconCircle} from '../../../components'
+
+export const LoginFirstCartScreen: FC<any> = ({navigation}) => {
+  const handleLogin = () => {
+    navigation.navigate('SignIn')
```

Comment:
good practice to have navigation screen names as a separate routes constants module (JFYI)

Reply:
Yeah, agree, it's a good practice. 
Will try to make such changes in the next iterations.